pkg/noderesourcetopology/pluginhelpers.go

```diff
@@ -0,0 +1,175 @@
+package noderesourcetopology
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+
+	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+	topoclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
+	topologyinformers "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/informers/externalversions"
+	listerv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/listers/topology/v1alpha1"
+)
+
+// topologyListerInstance is a Singleton object
+// Should not be accessed directly.
+// Should be accessed via getNodeTopologyInformer function
+var (
+	topologyListerInstance *listerv1alpha1.NodeResourceTopologyLister
```

Comment:
-1 on maintaining a global variable. IIRC, you want to ensure the informer to be instantiated once, but isn't that already the case (instantiated in New())?

Reply:
Yes. will remove that. it was useful when the plugins were separated. not needed anymore